modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/model-header.mustache

```diff
@@ -24,6 +24,61 @@ namespace {{this}} {
 {{#vendorExtensions.x-forward-declarations}}{{.}}
 {{/vendorExtensions.x-forward-declarations}}
 {{/vendorExtensions.x-has-forward-declarations}}
+{{#oneOf}}{{#-first}}
+#include <variant>
```

Comment:
We should move the include towards the top so that it is before the namespaces. Probably before or after `{{#imports}}`?